src/datepicker/datepicker.component.ts

```diff
@@ -55,21 +55,26 @@ export class DatePicker {
 
 	@Input() label: string;
 
-	@Input() label2: string;
+	@Input() rangeLabel: string;
 
 	@Input() placeholder = "mm/dd/yyyy";
 
 	@Input() pattern = "\d{1,2}/\d{1,2}/\d{4}";
 
 	@Input() id = `datepicker-${nextId++}`;
 
-	@Output() selectDates: EventEmitter<any> = new EventEmitter();
+	@Input() date: string;
+
+	@Output() datesChange: EventEmitter<any> = new EventEmitter();
```

Comment:
If it's `date` then it should be `dateChange` so we can double bind on `date`
```suggestion
	@Output() dateChange: EventEmitter<any> = new EventEmitter();
```